src/cocotb/types/logic_array.py

```diff
@@ -336,6 +336,15 @@ def from_bytes(
             int.from_bytes(value, byteorder=byteorder, signed=False), range=range
         )
 
+    @classmethod
+    def _from_handle(cls, value: str) -> "LogicArray":
```

Comment:
We cannot assert for it, but please mention in a comment that `value` is expected to be all-uppercase.